book/src/data/basics.md

```diff
@@ -20,56 +18,60 @@ explanation of the parameters in the [codebook](/codebook-ref.md#continuous)
 
 ### Categorical data
 
-Continuous columns are modeled as mixtures of [categorical
+> **Note:** For categorical columns, lace currently supports up to 256
+> unique values.
+
+Categorical columns are modeled as mixtures of [categorical
 distributions](https://en.wikipedia.org/wiki/Categorical_distribution). Find an
 explanation of the parameters in the [codebook](/codebook-ref.md#categorical). 
 
 ### Count data
 
 Support exists for a count data type, which is modeled as a mixture of
-[Poission distirbutions](https://en.wikipedia.org/wiki/Poisson_distribution),
+[Poission distributions](https://en.wikipedia.org/wiki/Poisson_distribution),
 but there are some drawbacks, which make it best to convert the data to
-continuous.
+continuous in most cases.
 
 - The Poisson distribution is a single parameter model so the location and
     variance of the mixture components cannot be controlled individually. In
     the Poisson model, higher magnitude means higher variance.
 - The hyper prior for count data is finicky and can often cause
-    underflow/overflow errors when the underlying data do not look like Poisson
-    distributions.
+    underflow/overflow errors (panics) when the underlying data do not look
```

Comment:
Would it be worth qualifying this as a "Rust panic", or something else? I don't know if the word "panic" has meaning to Python devs.